plugins/redfish/fu-redfish-backend.c

```diff
@@ -250,7 +267,15 @@ fu_redfish_backend_coldplug(FuBackend *backend, FuProgress *progress, GError **e
 			return FALSE;
 		}
 	}
-	if (json_object_has_member(json_obj, "MultipartHttpPushUri")) {
+	if (self->push_uri_path == NULL) {
```

Comment:
does Supermicro have `MultipartHttpPushUri` too? If not it might be good to move this to the last case, so that we start using the standard device types if Supermicro ever exports *both*.

Reply:
It does. No idea if it works, though. Their documentation doesn't mention it.

Reply:
Okay, maybe just add a comment to this effect; i.e. why it's listed first please

Reply:
Ok, I messed it up. Supermicro also uses `MultipartHttpPushUri` but needs an additional POST to `UpdateService.StartUpdate` to do the actual update. I'll update the code.

Reply:
Thanks!